packages/angular_devkit/core/node/host.ts

```diff
@@ -91,24 +91,7 @@ export class NodeJsAsyncHost implements virtualFs.Host<Stats> {
     return this.isDirectory(path).pipe(
       mergeMap(async (isDirectory) => {
         if (isDirectory) {
```

Comment:
With the use of only `rm`, the `isDirectory` check should also be able to be removed.

Reply:
Done